### PR TITLE
enh(broker) add pool size conf

### PR DESCRIPTION
**Pull request by loiclau**
## Description

add pool size broker configuration 

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by loiclau** [comment]
wait update script to add 

ALTER TABLE `cfg_centreonbroker` ADD COLUMN `pool_size` INT(11) DEFAULT NULL AFTER `daemon`;

File: www/include/configuration/configCentreonBroker/DB-Func.php [hunk 1]
```diff
@@ -121,24 +121,25 @@ function getCentreonBrokerInformation($id)
 
     $query = "SELECT config_name, config_filename, ns_nagios_server, stats_activate,
                     config_write_timestamp, config_write_thread_id, config_activate, event_queue_max_size,
-                    cache_directory, command_file, daemon
-                  FROM cfg_centreonbroker 
+                    cache_directory, command_file, daemon, pool_size
+                  FROM cfg_centreonbroker
                   WHERE config_id = " . $id;
```
Comment (sc979) [inline www/include/configuration/configCentreonBroker/DB-Func.php:126]: odd indentation

**Review by sc979 — DISMISSED** [review]

**Review by sc979 — APPROVED** [review]

**Comment by sc979** [comment]
you should maybe do another PR to reflect the SQL script on 21.04-beta.1 too, using IF NOT EXISTS

**Comment by kduret** [comment]
> you should maybe do another PR to reflect the SQL script on 21.04-beta.1 too, using IF NOT EXISTS

I will not make an update script for 21.04 since it has not been released yet

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**